src/js/components/CharityPage.jsx

```diff
@@ -136,27 +128,78 @@ const CharityAnalysisSection = ({charity}) => (
 	</div>
 );
 
+//this function gets project data of a certain charity, pushes the most recent projects to the outputarray and returns the total cost
```

Comment:
Let's document this a bit more, as the outputarray data structure is buried in the middle of the code.

Reply:
Thanks Aishah. I'd like to doc the data-types e.g. Is the returned total cost a Number or a Money object? 
(Hm: looks like it's neither -- maybe use Money?)
outputarray is not Project[], it's {color,title,value}[]